tests/collector.py

```diff
@@ -9,7 +9,7 @@ def test_parse_indexes(self):
     oids = {(1, 2, 3, 4): 'eth0', (1, 3, 65, 32, 255): 'octet',
             (1, 4, 10, 181, 1, 30): '10.181.1.30',
```

Comment:
This is missing the length, use a different value to make sure you're testing the right thing.